exchangelib/util.py

```diff
@@ -291,7 +301,7 @@ def post_ratelimited(protocol, session, url, headers, data, timeout=None, verify
             try:
                 r = session.post(url=url, headers=headers, data=data, allow_redirects=False, timeout=timeout,
                                  verify=verify)
-            except (requests.exceptions.ChunkedEncodingError, requests.exceptions.ConnectionError, ConnectionResetError,
+            except (requests.exceptions.ChunkedEncodingError, requests.exceptions.ConnectionError,
```

Comment:
ConnectionResetError is a real exception in Python3. Can you conditionally import this one?